tools/sandbox/application.py

```diff
@@ -152,30 +152,27 @@ def analyze(self) -> None:
             text=True,
         ) as process:
             model_verification_errors = []
-            model_verification_errors_found = False
             for line in iter(process.stderr.readline, b""):
                 line = line.rstrip()
                 if line == "":
                     break
-                elif "DEBUG" in line:
-                    continue
                 elif "ERROR" in line and "is not part of the environment" in line:
                     model_verification_errors.append(line)
-                    model_verification_errors_found = True
-                    continue
-                elif model_verification_errors_found:
-                    # Emit all model verification lines together to prevent
-                    # network overhead.
-                    emit(
-                        "pysa_results_channel",
-                        {
-                            "type": "output",
-                            "line": "\n".join(model_verification_errors),
-                        },
-                    )
-                    LOG.debug("\n".join(model_verification_errors))
+                elif "INFO" in line or "ERROR" in line:
+                    if any(model_verification_errors):
```

Comment:
Nit: Is there a reason why we are using `any` instead of `len(model_verification_errors)` here?

```suggestion
                    if len(model_verification_errors) > 0:
```

Reply:
Hey yes! So @abishekvashok sent the snippet and it seemed to be working so it slipped by me... I'll remove it...